Classes/Cache/DatabaseCache.php

```diff
@@ -177,15 +168,6 @@ public function getUrlFromCacheBySpeakingUrl($rootPageId, $speakingUrl) {
 			$requestVariables = json_decode($row['request_variables'], TRUE);
 			// TODO Log a problem here because it must be an array always
 			$cacheEntry->setRequestVariables(is_array($requestVariables) ? $requestVariables : array());
-
```

Comment:
I didnt find any relevant code, where you need the tstamp value. but you always make an update. why?